anymatrix.m

```diff
@@ -310,6 +310,40 @@
         IDs = IDs.';
     end
 
+
+    % Try to run function setup() if current group has one.
+    function run_setup_function_if_repository_changed(changed)
+        run_function_if_repository_changed(changed, 'setup');
+    end
+
+    % Try to run function update() if current group has one.
+    function run_update_function_if_repository_changed(changed)
+        run_function_if_repository_changed(changed, 'update');
+    end
+
+    % If current group has a function called `function_name`, run it after
+    % asking user for confirmation.
+    function run_function_if_repository_changed(changed, function_name)
+        if changed && exist([function_name '.m'], 'file')
```

Comment:
Good point. This is currently not necessary because `run_function_if_repository_changed` is only called here
```
old_folder = cd(strcat(group_folder, 'private/'));
run_setup_function_if_repository_changed(true);
cd(old_folder);
```
and here
```
old_folder = cd(strcat(group_folder, 'private/'));
[exit_code, output] = system('git pull', '-echo');
repository_changed = ~strcmp(output,...
                             ['Already up-to-date.', newline]);
run_update_function_if_repository_changed(repository_changed);
cd(old_folder);
```
so that `<function_name>.m` is in the current directory and therefore on the MATLAB path. But this is definitely not clean code—I'll fix it.